bulk-move.php

```diff
@@ -66,6 +66,7 @@ function bm_autoloader( $class_name ) {
 function load_bulk_move() {
 	$bulk_move = bulk_move();
 	$bulk_move->set_plugin_file( __FILE__ );
+	$bulk_move->add_loadie( new BM_Request_CustomTaxonomyAction() );
```

Comment:
The Ajax handlers should be the responsibility of the Metabox, not the main plugin class.

Reply:
Once the Ajax handlers are moved to the Metabox module, the Loadie interface will not be needed. Let's move that code to a new branch and keep it.

If we ever need a Loadie interface then we can merge it into master.

Reply:
@Sudar,

> The Ajax handlers should be the responsibility of the Metabox, not the main plugin class.

This is so true. I initially added the AJAX requests as part of the metabox, and found that the response from the AJAX request was always 0. I found that this was because that the callback couldn't be found by the `admin-ajax.php` file.

I'll move back the code and make a commit, for you to refer.